openedx/adg/lms/applications/admin.py

```diff
@@ -11,10 +12,11 @@ class ApplicationHubAdmin(admin.ModelAdmin):
     """
     Django admin class for ApplicationHub
     """
-    fields = ('user', 'is_prerequisite_courses_passed', 'is_application_submitted', 'is_assessment_completed',)
+    fields = ('user', 'is_prerequisite_courses_passed', 'is_written_application_completed',
+              'is_application_submitted', 'submission_date')
```

Comment:
I think you will get conflicts with master and develop. I have checked master is_assessment_completed is already removed by javeria. Please rebase your branch from latest master. You will not have to change this model after that. Do revert your migrations before getting latest pull. After getting latest pull add your fields in application hub. Also have a sync up with @harris-irfan so that he is not adding such fields or updating this admin site.
@harris-irfan bhai please have a look at the munam's changes and Please suggest as per your changes.

Reply:
@asadali145 I'm not changing anything on ApplicationHub model.